src/include/rados/librados.h

```diff
@@ -2642,6 +2642,36 @@ CEPH_RADOS_API int rados_notify2(rados_ioctx_t io, const char *o,
 				 uint64_t timeout_ms,
 				 char **reply_buffer, size_t *reply_buffer_len);
 
+/**
+ * Decode a notify message
+ *
+ * Decode a notify message (from rados_aio_notify() call) into acks and
+ * timeout arrays.
+ *
+ * @param reply_buffer buffer from rados_aio_notify() call
+ * @param reply_buffer_len reply_buffer length
+ * @param acks pointer to struct notify_ack_t pointer
+ * @param nr_acks pointer to ack count
+ * @param timeouts pointer to notify_timeout_t pointer
+ * @param nr_timeouts pointer to timeout count
+ * @returns 0 on success
+ */
+CEPH_RADOS_API int rados_decode_notify_msg(char *reply_buffer, size_t reply_buffer_len,
```

Comment:
Nit: should these methods actually be named something like `...decode_notify_response` or `...decode_notify_result` since this is not decoding the notification message but rather the result.